sdk/core/Azure.Core.Expressions.DataFactory/tests/DataFactoryElementTests.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace Azure.Core.Expressions.DataFactory.Tests
 {
-    public class DataFactoryExpressionsTests
```

Comment:
Can we add test cases around the other kinds?

Reply:
Yes, will need to add tests before marking as ready for review.